src/business-logic/services/database-change-ingest-service.ts

```diff
@@ -104,24 +243,40 @@ export class DatabaseChangeIngestService implements IngestChangeService {
       })
   }
 
+  private getEventToExecute(): (() => Promise<void>) | undefined {
+    const events: (() => Promise<void>)[] | undefined = Object.entries(this.eventPriorityQueue)
+      .sort(([a], [b]) => Number.parseInt(a) - Number.parseInt(b))
```

Comment:
`a` and `b` can be renamed to `priorityA` and `priorityB` to indicate that it is the priority that is sorted by.

You can consider using a Map instead of a record. That way priorities will keep their number type, when retrieving the entries.

Another solution will be to implement/use a max heap – but that is outside the scope of this PR :D

Reply:
Updated the names.
I'm more than open for alternatives for how to handle this, but it's going to be in another PR :)